CodeReviewGabe.cpp

```diff
@@ -0,0 +1,276 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <algorithm>
+#include <string>
+using namespace std;
+
+class User {
+  public:
+    string username;
+    string displayName;
+    string state;
+    string friends[10];
+};
+
+class Post {
+  public:
+    string postID;
+    string userID;
+    string visibility;
+};
+
+void splitString(string& input, char delimiter, 
+                 string arr[], int& index) 
+{ 
+    // Creating an input string stream from the input string 
+    istringstream stream(input); 
+
+    // Temporary string to store each token 
+    string token; 
+
+    // Read tokens from the string stream separated by the 
+    // delimiter 
+    while (getline(stream, token, delimiter)) { 
+        // Add the token to the array 
+        arr[index++] = token; 
+    } 
+} 
+
+int main() {
+  User user1;
+  User user2;
+  User user3;
+
+  Post post1;
+  Post post2;
+  Post post3;
```

Comment:
Is this going to work if a test case has 4 (or 2) posts? Storing these in a vector/array would also mean you could use a loop instead of duplicating code later